### PR TITLE
Add 1.15.5 release notes

### PR DESCRIPTION
**Pull request by Monkeyanator**
Planned release Monday, 1/30.

### CODE REVIEW

File: content/en/news/releases/1.15.x/announcing-1.15.5/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+{{< relnote >}}
+
+# Changes
+
+- **Added** the `--revision` flag to `istioctl analyze` making it possible to specify a revision.
+  ([Issue #38148](https://github.com/istio/istio/issues/38148))
+
+- **Fixed** a bug causing the abnormal exit of istiod when DestinationRule `PortLevelSettings[].Port` was nil. ([Issue #42598](https://github.com/istio/istio/issues/42598))
+
+- **Fixed** a bug causing namespace level network labels (`topology.istio.io/network`) to take precedence over workload level ones. ([Issue #42675](https://github.com/istio/istio/issues/42675))
```
Comment (kfaseela) [inline content/en/news/releases/1.15.x/announcing-1.15.5/index.md:23]: should we mention https://github.com/istio/istio/pull/42968 as well?

Comment (ericvn) [reply content/en/news/releases/1.15.x/announcing-1.15.5/index.md:23]: The text for this seems to mention workload while I think a similar item in the 1.16.2 RN mention pod.

Comment (kfaseela) [reply content/en/news/releases/1.15.x/announcing-1.15.5/index.md]: Thanks! LGTM... leaving it to @ericvn to merge!

File: content/en/news/releases/1.15.x/announcing-1.15.5/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+## Changes
+
+- **Added** the `--revision` flag to `istioctl analyze` making it possible to specify a revision.
+  ([Issue #38148](https://github.com/istio/istio/issues/38148))
+
+- **Fixed** a bug causing the abnormal exit of istiod when DestinationRule `PortLevelSettings[].Port` was nil. ([Issue #42598](https://github.com/istio/istio/issues/42598))
+
+- **Fixed** a bug causing namespace level network labels (`topology.istio.io/network`) to take precedence over pod labels. ([Issue #42675](https://github.com/istio/istio/issues/42675))
+
+- **Added** mitigation for a request smuggling vulnerability caused by an issue in the Go http2 library. ([Issue #56352](https://github.com/golang/go/issues/56352))
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15.5/index.md:25]: You should move this up next to the other Added.

Comment (Monkeyanator) [reply content/en/news/releases/1.15.x/announcing-1.15.5/index.md]: Done.

File: content/en/news/releases/1.15.x/announcing-1.15.5/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Istio 1.15.5
+linktitle: 1.15.5
+subtitle: Patch Release
+description: Istio 1.15.5 patch release.
+publishdate: 2023-1-30
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15.5/index.md:6]: I think we need to make this 2023-01-30 as when looking at the deploy output, I do not see the news item.

**Review by ericvn — APPROVED** [review]

**Pull request merged**